crates/biome_js_analyze/src/lint/complexity/no_useless_fragments.rs

```diff
@@ -401,3 +415,9 @@ impl Rule for NoUselessFragments {
         }))
     }
 }
+
+fn contains_html_entity(s: &str) -> bool {
```

Comment:
These are called "Character references" in the HTML spec

Reply:
ok~ i update the util name